script/release/publish-to-npm.js

```diff
@@ -175,6 +175,10 @@ new Promise((resolve, reject) => {
             semver.gt(localVersion, currentTags.beta)) {
         childProcess.execSync(`npm dist-tag add electron@${localVersion} beta --otp=${process.env.ELECTRON_NPM_OTP}`);
       }
+      if (parsedLocalVersion.prerelease[0] === 'alpha' &&
+            semver.gt(localVersion, currentTags.alpha)) {
+        childProcess.execSync(`npm dist-tag add electron@${localVersion} alpha --otp=${process.env.ELECTRON_NPM_OTP}`);
```

Comment:
We also have to make changes on line 140~ish to make the tag `alpha-${currentBranch}`

Reply:
I added a check for the alpha NPM tag on 138:
```
} else if (release.tag_name.indexOf('alpha') > 0) {
```

I double-checked the GitHub tag system, but I'm a little wary of whether or not this check will work as expected.